grpc-sys/grpc_wrap.cc

```diff
@@ -57,6 +57,9 @@
 #include <grpc/support/string_util.h>
 #include <grpc/support/thd_id.h>
 
+#include "src/core/lib/gpr/useful.h"
+#include "src/core/lib/iomgr/socket_mutator.h"
```

Comment:
This will prevent users to link to local grpc library. Only public headers can be depended on.

Reply:
Ah, thanks, this should be accessible through `grpc-sys/grpc/include/grpc/impl/codegen/grpc_types.h` instead.

Reply:
On a more detailed review, `grpc_socket_mutator` is a forward decl in grpc_types.h, however `grpc_socket_mutator_vtable` is not, and so part of the required api is not public.